src/main/java/com/jfrog/ide/idea/ui/ComponentsTree.java

```diff
@@ -71,35 +73,24 @@ private void handleContextMenu(ComponentsTree tree, MouseEvent e) {
         if (!e.isPopupTrigger()) {
             return;
         }
-
         // Event is right-click.
-        TreePath selPath = tree.getPathForLocation(e.getX(), e.getY());
-        if (selPath == null) {
+        TreePath selectedPath = tree.getPathForLocation(e.getX(), e.getY());
+        if (selectedPath == null) {
             return;
         }
-        createNodePopupMenu((DependencyTree) selPath.getLastPathComponent());
-        popupMenu.show(tree, e.getX(), e.getY());
+        if (selectedPath.getLastPathComponent() instanceof DependencyNode) {
+            createNodePopupMenu((DependencyNode) selectedPath.getLastPathComponent());
+            popupMenu.show(tree, e.getX(), e.getY());
```

Comment:
For better user experience - let's show the menu when clicking not only on the component text, but also in the right of it